inference-gateway/migrations/20240626175841_fdw.sql

```diff
@@ -0,0 +1,29 @@
+CREATE EXTENSION IF NOT EXISTS postgres_fdw;
+CREATE EXTENSION IF NOT EXISTS pg_cron;
+
+CREATE SERVER IF NOT EXISTS foreign_validation_server
+FOREIGN DATA WRAPPER postgres_fdw
+OPTIONS (host 'localhost', port '5432', dbname 'postgres');
+
+CREATE USER MAPPING IF NOT EXISTS FOR postgres
+SERVER foreign_validation_server
+OPTIONS (user 'postgres', password 'postgres');
+
+CREATE FOREIGN TABLE IF NOT EXISTS fdw_paid_organizations (
```

Comment:
It seems problematic in my opinion to work this way because the data plane should not have access to the control plane db at all. For example it would cause problems with installing this into a self hosted situation.

Reply:
self hosted installations will not use organization level validation nor token-based billing models. they will set `ORG_VALIDATION_ENABLED=false`

Reply:
Adam mentioned on Slack we can configure this to connect to the queue DB, not the control plane DB